### PR TITLE
Ember optional

### PR DESCRIPTION
**Pull request by iezer**
2nd commit makes ember and glimmer optional based on if they are in the config. Probably not a great long term solution but for now it works for both of our use cases. Merge after #9

### CODE REVIEW

File: lib/utils.rb [hunk 1]
```diff
@@ -178,14 +179,20 @@ def in_dir(path, expand: true, silent: false)
...
   end
 
   def each_experiment
-    @experiments ||= parse_json("config/experiments.json")
-    @experiments.each { |experiment| yield experiment }
+    experiments.each { |experiment| yield experiment }
+  end
+
+  def custom_ember_glimmer
+    @custom_ember_glimmer ||= experiments.all? { |e| e["ember"] && e["glimer"] }
```
Comment (rwjblue) [inline lib/utils.rb]: Lets base this off of `e['ember']` only, inside the `build_ember` script I am going to make the glimmer build optional separately

**Review by rwjblue — CHANGES_REQUESTED** [review]
Looking good!  I submitted #11 which is complementary to this (allows building Ember without building a custom Glimmer).

File: lib/utils.rb (hunk 1 above)
Comment (iezer) [reply lib/utils.rb]: I made this change and also updated the README.md which reflects this and your other PR #11

**Comment by rwjblue** [comment]
Thanks @iezer!  We merged this in https://github.com/chancancode/ember-bench/commit/312fdd5c500f5b5d343ad0e14b20f765a0a36fb3....

**Pull request closed without merging**